config/examples/Creality/Ender-3/BigTreeTech SKR E3-DIP V1.1/Configuration.h

```diff
@@ -1290,7 +1281,7 @@
 #if EITHER(AUTO_BED_LEVELING_LINEAR, AUTO_BED_LEVELING_BILINEAR)
 
   // Set the number of grid points per dimension.
-  #define GRID_MAX_POINTS_X 5
+  #define GRID_MAX_POINTS_X 3
```

Comment:
This is for restoring default value, because the parameter has no meaning without BLtouch (forgotten).